assets/js/sections/contesting.js

```diff
@@ -208,26 +213,43 @@ $('#start_date').change(function () {
 
 // On Key up check and suggest callsigns
 $("#callsign").keyup(function () {
-	var call = $(this).val();
+	let call = $(this).val().toUpperCase();
 	if (call.length >= 3) {
 
-		$.ajax({
-			url: 'lookup/scp',
-			method: 'POST',
-			data: {
-				callsign: $(this).val().toUpperCase()
-			},
-			success: function (result) {
-				$('.callsign-suggestions').text(result);
-				highlight(call.toUpperCase());
-			}
-		});
+        if ( scp_data.request == "" || ! call.startsWith(scp_data.request) ) {
+          scp_data.request = call;
+          scp_data.data = [];
+          $.ajax({
+              url: 'lookup/scp',
+              method: 'POST',
+              data: {
+                  callsign: call
+              },
+              success: function (result) {
+                  var call_now = $("#callsign").val().toUpperCase(); // Might have changed while running the query...
```

Comment:
this should be `let`

Reply:
`var` and `let` are equivalent here.

Reply:
no, they arent.  please read about javascript before submitting PR's to change javascript.